README.md

```diff
@@ -73,12 +73,13 @@ npm install -D @types/node
 
 ## Troubleshooting
 
-The MongoDB driver depends on several other packages. These are:
+The MongoDB driver can optionally be enhanced by the following feature packages:
 
-- [bson](https://github.com/mongodb/js-bson)
-- [bson-ext](https://github.com/mongodb-js/bson-ext)
-- [kerberos](https://github.com/mongodb-js/kerberos)
-- [mongodb-client-encryption](https://github.com/mongodb/libmongocrypt#readme)
+- [Zstd network compression](https://github.com/mongodb-js/zstd)
```

Comment:
I think listing the package names is still worthwhile, if we want to add a short description like we have currently, we can do that, too, but I think it's more convenient for users to reference them by the name they can `npm install` (or look up in the npm repository). I am also not sure that we should be listing packages we don't own, and if we do list them, then we should clearly label them as 3rd party. Given that they are in the troubleshooting section, I'm leaning toward not including them at all.

Reply:
Yea this section was a bit off, bson was listed and it's not a c++ dep nor is it something that needs troubleshooting (I hope 😅) 

Updated this section, and I put the link to the troubleshooting under our dependencies only